### PR TITLE
Create account w/ initial deposit: handle error

### PR DESCRIPTION
**Pull request by Patrick1904**

### CODE REVIEW

**Comment by Patrick1904** [comment]
@MaximusHaximus I added similar error handling to account creation with initial deposit as we added for [coin op](https://github.com/near/near-wallet/pull/2237). PTAL when you have a chance.

File: packages/frontend/src/components/accounts/create/initial_deposit/InitialDepositWrapper.js [hunk 1]
```diff
@@ -99,9 +101,55 @@ export function InitialDepositWrapper({ history }) {
...
+                    await wallet.saveAndMakeAccountActive(accountId);
+                    await wallet.addLocalKeyAndFinishSetup(accountId, recoveryMethod, publicKey);
+                } catch (e) {
+                    dispatch(showCustomAlert({
+                        success: false,
+                        messageCodeHeader: 'error',
+                        messageCode: 'walletErrorCodes.createNewAccount.accountCreated.error',
+                        errorMessage: e.message
+                    }));
+                    redirectTo('/recover-account');
```
Comment (MaximusHaximus) [inline packages/frontend/src/components/accounts/create/initial_deposit/InitialDepositWrapper.js:147]: Should this be using a `dispatch()`?

**Review by MaximusHaximus — CHANGES_REQUESTED** [review]
:+1: Just one question about not using `dispatch()` for the `redirectTo` action

File: packages/frontend/src/components/accounts/create/initial_deposit/InitialDepositWrapper.js (hunk 1 above)
Comment (Patrick1904) [reply packages/frontend/src/components/accounts/create/initial_deposit/InitialDepositWrapper.js]: Great catch:)

**Pull request merged**